data/json/professions.json

```diff
@@ -6265,5 +6305,605 @@
+          { "item": "lye_powder", "charges": 200, "container-item": "bag_plastic" },
+          { "item": "chem_acetone", "charges": 15, "container-item": "jug_plastic", "custom-flags": [ "auto_wield" ] },
+          { "item": "chem_methanol", "charges": 250, "container-item": "bottle_plastic_small" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "sommelier",
+    "name": "Sommelier",
+    "description": "You have an undying passion for wine.  Not even the end of the world is going to stop you from degustating more kinds of it.  Hell, perhaps it's time to make some of your own.",
+    "points": 3,
+    "skills": [ { "level": 4, "name": "cooking" }, { "level": 3, "name": "chemistry" } ],
+    "traits": [ "GOURMAND" ],
+    "proficiencies": [ "prof_beverage_distilling", "prof_fermenting", "prof_winemaking" ],
+    "items": {
+      "both": {
+        "items": [ "suit", "dress_shoes", "socks" ],
+        "entries": [ { "group": "charged_smart_phone" }, { "item": "wine_noir", "custom-flags": [ "auto_wield" ] } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "contract_sniper",
+    "name": "Contract Sniper",
+    "requirement": "achievement_kill_100_monsters",
+    "description": "You were the best sniper money could buy.  Many criminal enterprises hired you to take out VIPs, this time was no different.  Your target seems to have turned feral as well as your contact, you are all alone now.",
+    "points": 6,
+    "skills": [
+      { "level": 8, "name": "gun" },
+      { "level": 8, "name": "rifle" },
+      { "level": 4, "name": "pistol" },
+      { "level": 4, "name": "speech" }
+    ],
+    "proficiencies": [ "prof_gunsmithing_basic", "prof_spotting", "prof_wp_syn_armored", "prof_gun_cleaning" ],
+    "items": {
+      "both": {
+        "items": [
+          "fancy_sunglasses",
+          "platinum_watch",
+          "polaroid_photo",
+          "dress_shirt",
+          "dress_shoes",
+          "socks",
+          "hat_cotton",
+          "tie_skinny"
+        ],
+        "entries": [
+          { "group": "charged_cell_phone" },
+          { "item": "pants", "variant": "pants_black", "contents-group": "ccp_mags" },
+          {
+            "item": "weatherby_5",
+            "ammo-item": "300_winmag",
+            "charges": 3,
+            "contents-item": [ "shoulder_strap", "suppressor", "bipod", "rifle_scope" ]
+          },
+          { "item": "300_winmag", "charges": 20 },
+          { "item": "walther_ccp", "ammo-item": "9mmfmj", "charges": 8, "container-item": "holster" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    },
+    "missions": [ "MISSION_ASSASSINATION" ]
+  },
+  {
+    "type": "profession",
+    "id": "carpenter",
```

Comment:
This doesn't really provide anything distinctive in addition to the crafting hobby, part of the point of hobbies is to reduce clutter for skillset-specific professions.

Reply:
I disagree on this one, but it should maybe have a few more associated proficiencies first